api/src/docs.rs

```diff
@@ -883,7 +883,7 @@ impl HrefResolver for DocResolver {
     let (is_main, path) = current_resolve
       .get_file()
       .map(|short_path| (short_path.is_main, &*short_path.path))
-      .unwrap_or_default();
+      .unwrap_or((true, ""));
```

Comment:
Should we add a test for this?

Reply:
we need to rework the enture usage block infrastructure in the next few days anyways, so i think lets just land as-is